taxonomy.simple.yaml

```diff
@@ -36,14 +36,18 @@
 
   - Education:
     - Afterschool Care
-    - Respite Activities:
-      - Further Education:
-        - Engineering
-        - Yeshiva and Kollel
-      - Academia:
+    - Nursery
+    - Respite Activities
+    - Further Education
```

Comment:
missing :

Reply:
No missing, the : was a mistake from previous pull request! :-0